docs/tutorial/t-overview.md

```diff
@@ -1,26 +1,20 @@
-# Charmed Spark Tutorial 
+## Charmed Spark Solution Tutorial
 
 The Charmed Spark solution delivers Spark utility client applications, that allow for simple and seamless usage of Apache Spark on Kubernetes.
 
-First this tutorial provides instructions on how to get a simple Kubernetes distribution up and running. Then it describes how to run Spark Jobs, either as a separate process or interactively. 
+This tutorial takes you through the journey of setting up necessary environment to run Charmed Spark and teaches you how to run Spark jobs using both interactive shell and batch jobs. You'll also learn how to use Charmed Spark with streaming workloads and how to enable monitoring of the Spark cluster.
 
-Through this tutorial you will learn a variety of operations, ranging from service account management, to actual computation. Finally we also provide a list of tips and tricks of common problems you may face when using Spark on Kubernetes. 
+This tutorial can be divided into the following sections:
+1. Overview
+2. Setting up environment for the tutorial
+3. Interacting with Spark using interactive shells
+4. Submitting Jobs with Spark Submit
+5. Streaming workload with Charmed Spark
+6. Monitoring the Spark cluster
+7. Wrapping Up
 
-This tutorial has been split into the following sections:
 
-1. [Setting up your environment with MicroK8s](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8951)
-2. [Setup and manage Spark service accounts](https://discourse.charmhub.io/t/spark-client-snap-tutorial-setup-environment/8952)
-3. [Submit a Spark Job using an enhanced version of `spark-submit`](https://discourse.charmhub.io/t/spark-client-snap-tutorial-spark-submit/8953)
-4. [Perform a simple analysis using `spark-shell` using Scala, and `pyspark` using Python](https://discourse.charmhub.io/t/spark-client-snap-tutorial-interactive-mode/8954)
-5. [Review a list of common tips, tricks and issues you may face when using Spark on Kubernetes](https://discourse.charmhub.io/t/spark-client-snap-tutorial-common-gotchas/8955)
+While this tutorial intends to guide and teach you along the way, it will be most beneficial if you already have a familiarity with:
+* Basic terminal commands.
```

Comment:
Who do we want to target with this tutorial? As someone completely inexperienced with K8s and Spark, I'd say that it's worth encouraging users to go through basic Kubernetes concepts if they have the time.

This didn't stop me from being able to follow along, so it's not strictly mandatory, but K8s concepts eventually get mixed in with Juju concepts (namespaces vs models vs controllers vs pods vs units...) making it all the more complex.

Reply:
Since Kubernetes is a platform on top of which our solution runs, I think it is better to add a general familiarity with K8s as requirements as suggested by @deusebio. I'm adding that line.